apps/reactotron-app/src/renderer/App.tsx

```diff
@@ -28,6 +29,22 @@ const AppContainer = styled.div`
   background-color: ${(props) => props.theme.background};
 `
 
+// This wrapper container is used to track page views within the app automatically using react-router-dom
+// as well as to initialize analytics on app load.
+const AppContainer: React.FC<{ children: React.ReactNode }> = ({ children }) => {
+  usePageTracking()
+  const { initializeAnalytics } = useAnalytics()
+
+  useEffect(() => {
+    const timer = setTimeout(() => {
+      initializeAnalytics()
+    }, 250)
```

Comment:
What are we waiting for here?

Reply:
While developing, I found that there was the possibility of this component rendering a couple times. This delay with the clearTimeout is meant to only initialize the analytics once the electron renderer process has completely opened the app and react has stopped reacting. Not sure if there's a better way to handle this other than some memoizing